docker/dash/Makefile

```diff
@@ -0,0 +1,38 @@
+.PHONY: build build-no-cache start stop logs rm reload all
+.DEFAULT_GOAL := all
+
+BIN=docker-compose -f docker-compose.yml
+
+help:
+	@echo "Usage: make [OPTION] [ENV=dev|test] [C=\"container\"]"
+	@echo ""
+	@echo "Options"
+	@echo "\thelp      		-- print this help"
+	@echo "\tbuild     		-- build docker environment"
+	@echo "\tbuild-no-cache   	-- build docker environment by invalidating cache"
+	@echo "\tstart        		-- start docker stack"
+	@echo "\tstop      		-- stop docker stack"
+	@echo "\tlogs       		-- get logs from docker stack"
+	@echo "\trm     			-- stop docker stack & clean all artifacts"
```

Comment:
I was thinking in managing all projects with a single Makefile, but for now I wanted to avoid having to provide a `PROJECT` parameter